apps/events/models/invitation.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+
+from django.db import models
+from apps.core.models import AbstractBaseModel
+from apps.events.models import Event
+
+user_model = settings.AUTH_USER_MODEL
+
+
+class Invitation(AbstractBaseModel):
+    class Status(models.TextChoices):
+        ACCEPTED = "ACCEPTED"
+        PENDING = "PENDING"
+        DECLINED = "DECLINED"
+
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name="event")
+    sender = models.ForeignKey(user_model, on_delete=models.CASCADE, related_name="sender")
+    recipient = models.ManyToManyField(user_model, verbose_name="list_of_recipients")
+    status = models.CharField(max_length=10, choices=Status.choices, default="PENDING")
+    event_url = models.TextField(max_length=250, blank=True)
```

Comment:
Might be a place to use models.URLField

Reply:
we should update recipient foreign key, and (recipient sender event ) should have unique constraint